editor/js/reaction.js

```diff
@@ -401,7 +474,12 @@ function isEmptyMessage(obj) {
   return false;
 }
 
-// The template is a jQuery selector. The root may be a jQuery object.
+/**
+ * Adds an instance of `template` to the root node.
+ * @param {string} template A jQuery selector.
+ * @param {!Node} root A jQuery object.
+ * @return {!Node}
```

Comment:
Could add "The new copy of the template" to the return documentation

Reply:
Done.